docs/references/message-execution-properties.mdx

```diff
@@ -14,26 +14,26 @@ The [community conversation on security best practices](https://www.youtube.com/
 
 To interact with a canister's methods, there are two primary types of **calls** that can be used: [update](/references/ic-interface-spec.md#http-call) calls and [query](/docs/current/references/ic-interface-spec#http-query) calls.
 
-If the Rust CDK is used, these are usually annotated with `#[update]` or `#[query]`, respectively. In Motoko, updates are declared as `public func`, and queries us the dedicated keyword: `public query func`. ICP also supports additional [entry points](/references/ic-interface-spec.md#entry-points) such as heartbeats, timers, initialization or upgrade hooks.
+The call must be addressed to an entry point of a canister, which is called a **method**. If the Rust CDK is used, these are usually annotated with `#[update]` or `#[query]`, respectively. In Motoko, updates are declared as `public func`, and queries use the dedicated keyword: `public query func`. ICP also supports additional [entry points](/references/ic-interface-spec.md#entry-points) such as heartbeats, timers, initialization or upgrade hooks.
 
-A **message** is a set of consecutive instructions that a subnet executes for a canister. The code execution for any such entry point can be split into several messages if inter-canister calls are made. The following properties are essential:
+A **message execution** is a set of consecutive instructions that a subnet executes when a canister's entry point is invoked. The code execution for any such entry point can be split into several message executions if inter-canister calls are made. The following properties are essential:
 
-- **Property 1**: Only a single message is processed at a time per canister. Message execution is atomic and sequential, and never parallel.
+- **Property 1**: Only a single message execution is run at a time per canister. Message execution is atomic and sequential, and never parallel.
```

Comment:
Nit: We start the page stating that the benefit of asynchronous messaging is that "individual messages can be executed in parallel". Maybe we should qualify one or both of the statements.

Reply:
Yeah, that's the technically correct statement, I'll rephrase. A single canister's calls can still be concurrent.

Reply:
I think I addressed all the points now raised, so resolving this.